From 3826032039fff248e885ab814a91fc3158506766 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Fri, 12 Sep 2014 18:03:52 -0400
Subject: [powerpc] pseries: Enable PSTORE in pseries_defconfig

Message-id: <1410545655-205645-4-git-send-email-dzickus@redhat.com>
Patchwork-id: 94110
O-Subject: [RHEL7 PATCH 003/626] powerpc/pseries: Enable PSTORE in pseries_defconfig
Bugzilla: 1127366
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Steve Best <sbest@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: David Gibson <dgibson@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1127366

commit 99b308e3bbacc26509858ac78ebc7a0a1dfbb888
Author: Aruna Balakrishnaiah <aruna@linux.vnet.ibm.com>
Date:   Mon Jun 24 12:23:00 2013 +0530

    powerpc/pseries: Enable PSTORE in pseries_defconfig

    Since now we have pstore support for nvram in pseries, enable it
    in the default config. With this config option enabled, pstore
    infra-structure will be used to read/write the messages from/to nvram.

    Signed-off-by: Aruna Balakrishnaiah <aruna@linux.vnet.ibm.com>
    Acked-by: Michael Neuling <mikey@neuling.org>
    Signed-off-by: Benjamin Herrenschmidt <benh@kernel.crashing.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/powerpc/configs/pseries_defconfig b/arch/powerpc/configs/pseries_defconfig
index 41a830b..1d4b976 100644
--- a/arch/powerpc/configs/pseries_defconfig
+++ b/arch/powerpc/configs/pseries_defconfig
@@ -296,6 +296,7 @@ CONFIG_SQUASHFS=m
 CONFIG_SQUASHFS_XATTR=y
 CONFIG_SQUASHFS_LZO=y
 CONFIG_SQUASHFS_XZ=y
+CONFIG_PSTORE=y
 CONFIG_NFS_FS=y
 CONFIG_NFS_V3_ACL=y
 CONFIG_NFS_V4=y
-- 
1.7.1